images/build-integration/lib/darwin/run.sh

```diff
@@ -4,6 +4,7 @@
 bundleLocation=""
 targetFolder="crc-integration"
 junitFilename="integration-junit.xml"
+suiteTimeout="90m"
```

Comment:
is the `images/build-integration/lib/darwin/run.sh` and `images/build-integration/lib/linux/run.sh` the same script? maybe we can use a symlink to avoid editing two files

Reply:
done as per suggest on https://github.com/crc-org/crc/pull/4169/commits/7b60e2e8c05da57b7171d47ded1f4ced353de5f5